src/core/components/states/emptyState/emptyState.tsx

```diff
@@ -14,7 +14,7 @@ export const EmptyState: React.FC<IEmptyStateProps> = ({
     humanIllustration,
 }) => {
     const containerClassNames = classNames(
-        'grid w-[320px] md:w-[640px]', // Defaults
+        'grid w-full max-w-sm', // Defaults
```

Comment:
Should we also update the fixed widths below? The IllustrationHuman component for instance has a ficed width set to `400px` and it will probably overflow if the component is placed on a container that does not have enough space

Reply:
@cgero-eth thanks for this, so I agree having fixed widths is generally not great, but in this case the 400px is only for `md:` breakpoints, so I don't think it will overflow. For small devices it is 295 and given these are so specific I wasn't sure about changing them and breaking image ratio or something like that. What do you think?

Reply:
I would still remove these fixed widths and make sure ratio and UI still works fine. Even if 400px width is set only for md breakpoint, the component itself might be placed inside a small container on md breakpoint and still overflow

Reply:
Yeah makes sense. I removed the 400px, but left some others as the explicit widths helped on smaller devices as the human illustration has a large frame so using full width would actually make the image look quite small.

It now looks nice on all configurations and screen sizes

Reply:
🔫